akka-docs/rst/project/migration-guide-persistence-experimental-2.3.x-2.4.x.rst

```diff
@@ -178,3 +178,19 @@ persistent actor on the sending side.
 
 Read more about at-least-once delivery in the :ref:`documentation for Scala <at-least-once-delivery>` and 
 :ref:`documentation for Java <at-least-once-delivery-java>`.  
+
+Default persistence plugins
+===========================
+Previously default ``akka.persistence.journal.plugin`` was set to the LevelDB journal ``akka.persistence.journal.leveldb``.
+Now persistence extension will instead use in-memory journal plugin ``akka.persistence.journal.inmem`` as a default.
+This change was needed to decouple persistence extension from the the LevelDB dependency.
```

Comment:
You changed this in an earlier pull request, and I agree that this is the reason why we must change it.
However, I think it would be better to not have a default at all. I don't want to be responsible in case someone previously used leveldb as the default and then drops in 2.4 into production and it silently stores things in memory.

Users don't read docs or migration guides until there is a problem.